### PR TITLE
chore: update deps and test on node v7

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,44 +7,41 @@
...
-    "supertest": "^1.2.0",
-    "supertest-as-promised": "^3.2.0"
+    "supertest": "^2.0.1"
   },
   "engines": {
     "node": ">=4.0.0"
   },
   "scripts": {
-    "test": "npm run lint && npm run test-local",
+    "test": "npm run lint -- --fix && npm run test-local",
```
Comment (popomore) [inline package.json:30]: 别加 --fix

Comment (atian25) [reply package.json:30]: test 这里加是可以的, 是本地跑的. 在 ci 不会执行它.

**Comment by atian25** [comment]
@popomore 没啥问题的话就合了吧

**Comment by atian25** [comment]
1.0.0

**Pull request merged**